ng/manage.ts

```diff
@@ -1,1639 +1,1639 @@
-                    }
+        $scope.clearCreate = () => {
+            delete $scope.createModel.createdResourceName;
+            createEditor.customSetValue(stringify($scope.createMetaData));
+        };
 
-                    delete $scope.createError;
-                    var userObject = JSON.parse(createEditor.getValue());
-                    cleanObject(userObject);
-                    $scope.invoking = true;
-                    var selectedBranch = $scope.treeControl.get_selected_branch();
-                    userHttp({
-                        method: "POST",
-                        url: "api/operations",
-                        data: {
-                            Url: $scope.putUrl + "/" + resourceName,
-                            HttpMethod: "PUT",
-                            RequestBody: userObject,
-                            ApiVersion: $scope.apiVersion
-                        }
-                    }, () => {
-                        $scope.treeControl.collapse_branch(selectedBranch);
-                        if (selectedBranch.uid === $scope.treeControl.get_selected_branch().uid) {
-                            $scope.selectResourceHandler($scope.treeControl.get_selected_branch(), undefined);
-                            fadeInAndFadeOutSuccess();
+        $scope.invokeCreate = (event) => {
+            try {
+                var resourceName = $scope.createModel.createdResourceName;
+                if (!resourceName) {
+                    $scope.createError = syntaxHighlight({
+                        error: {
+                            message: "{Resource Name} can't be empty"
                         }
-                        $timeout(() => {
-                            $scope.expandResourceHandler(selectedBranch);
-                        }, 50);
-                    }, (err) => {
-                        $scope.createError = syntaxHighlight(err);
-                        fadeInAndFadeOutError();
-                    }, () => {
-                        $scope.invoking = false;
-                        $scope.loading = false;
-                    }, event);
-                } catch (e) {
-                    $scope.createError = syntaxHighlight({ error: "Error parsing JSON" });
+                    });
                     $scope.invoking = false;
                     $scope.loading = false;
+                    return;
                 }
-            };
 
-            function refreshContent() {
-                $scope.selectResourceHandler($scope.treeControl.get_selected_branch(), undefined);
-            };
-
-            $scope.enterDataTab = () => {
-                [responseEditor, requestEditor].map((e) => {
-                    if (e) {
-                        e.resize();
+                delete $scope.createError;
+                var userObject = JSON.parse(createEditor.getValue());
+                cleanObject(userObject);
+                $scope.invoking = true;
+              
```

Comment:
changed the func name and impl. this is the only change in this file. (other than removing some now unused functions and moving GETPSObjectFromJSON() to PowerShellScriptGenerator.ts file)